python/pyspark/sql/connect/plan.py

```diff
@@ -207,7 +240,7 @@ def print(self, indent: int = 0) -> str:
     def _repr_html_(self) -> str:
```

Comment:
Why is this method not removed?

Reply:
This wasn't removed because `LocalRelation` contains `data` that's PyArrow table, and this isn't printed out pretty together with our format out of the box. So it was manually overridden here.